API/Handlers/Implementations/OAuthHandler.cs

```diff
@@ -23,8 +25,8 @@ public class OAuthHandler(
     IPlayersRepository playerRepository,
     IUserRepository userRepository,
     IOsuClient osuClient,
-    IPasswordHasher<OAuthClient> clientSecretHasher
-    ) : IOAuthHandler
+    IPasswordHasher<OAuthClient> clientSecretHasher,
+    IOptions<IOsuConfiguration> osuConfiguration) : IOAuthHandler
```

Comment:
It is not being injected, some config should be updated ig

Reply:
Will investigate.

Reply:
fixed